client/src/ui/atoms/button/index.tsx

```diff
@@ -61,6 +53,7 @@ export const Button = ({
   state,
   value,
   children,
+  ...dashedProps
```

Comment:
Are we sure these are only dashed props? If not, we could rename to `extraProps` (or `extraAttrs`).